### DISCUSSION TITLE
Add Square/Cubic Roots :)

### DISCUSSION THREAD

**Discussion by 19akshansh** [prakhartiwari0/Arito#6] [Ideas]
You can add square & cubic roots as they're also core concepts of mathematics. From Heron's Formula till finding many answers, a knowledge of roots are required.. So if ARITO is a site to learn/practice math, My suggestion would be to add these :)

**Comment by 3xp11i** [comment]
> You can add square & cubic roots as they're also core concepts of mathematics. From Heron's Formula till finding many answers, a knowledge of roots are required.. So if ARITO is a site to learn/practice math, My suggestion would be to add these :)

Oh yeah definitely!
Actually, I started this project for kids younger than 10-11 years of age, and they are not taught those concepts yet.
But we can add them somehow if you could describe which type of questions should be there. 
And will it be useful to practice "finding out square and cube roots"?

**Reply by 19akshansh** [reply]
The type of question that can be there are as follows:
Square root of RandomNumber
Let RandomNumber be 144 in this case, it's square root would be 12.(12×12 = 1024).

Practicing square roots will help students in finding areas of triangles/squares much faster when they use HERON'S FORMULA.

But if the site is for kids younger than 11years, I don't think it should be implemented...

**Reply by 3xp11i** [reply]
The app is not specifically for kids younger than 11 years of age, I started it with the thought of keeping it in these 4 operators only (+, -, x, and /). But in the future, we can implement a lot more things like equations, graphical questions, geometry, and more (it would be done when the app is moving to an advanced level ).

Talking about square roots, I don't think it is something to practice, I mean if we are giving the user question for PERFECT SQUARE NUMBERS, they just need to memorize that stuff, and that too to a limit. It's just my point of view if you want to add any more points you are welcome.
